parquet-tools/src/main/java/org/apache/parquet/tools/command/MergeCommand.java

```diff
@@ -46,19 +47,34 @@
           "   <output> is the destination parquet file"
   };
 
-  private static final String DEFAULT_KEY_VALUE_MERGE_STRATEGY = StrictKeyValueMetadataMergeStrategy.class.getName();
+  private static enum MergeStrategy {
+    STRICT,
+    CONCAT,
+    CUSTOM,
+  }
+  
   public static final Options OPTIONS;
 
   static {
     OPTIONS = new Options();
     Option mergeStrategy = Option.builder("s")
       .longOpt("mergeStrategy")
       .desc("Strategy to merge (key, value) pairs in metadata if there are multiple values for same key " +
-        "(default: " + DEFAULT_KEY_VALUE_MERGE_STRATEGY + ")")
+        "(default: " + MergeStrategy.STRICT + "). You can provide your custom implementation by specifying " +
```

Comment:
We should list the available options here. (The best way would be do it by iterating over the enum so a potential extension later would not forget about this part.) 

nit: in command line we rarely use upper case arguments. Maybe it would be nicer to convert the enum names to lower case.

Reply:
Ok. Done.